lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -4653,11 +4653,14 @@ namespace Js
         if (!IsIntermediateCodeGenThunk(defaultEntryPointInfo->jsMethod)
             && defaultEntryPointInfo->jsMethod != DynamicProfileInfo::EnsureDynamicProfileInfoThunk)
         {
+#ifdef ENABLE_SCRIPT_PROFILING
```

Comment:
Is this needed? This function looks like it's already wrapped under #ifdef ENABLE_SCRIPT_PROFILING